pkg/cmd/config/config.go

```diff
@@ -0,0 +1,149 @@
+package config
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+)
+
+type Config struct {
+	LogLevel  string `json:"log_level"`
+	Progress  string `json:"progress"`
+	ConfigDir string `json:"config_dir"`
+	Verbose   int    `json:"verbose"`
+}
+
+// DefaultConfig returns a Config struct with default values.
+func DefaultConfig() *Config {
+	return &Config{
+		LogLevel:  "info",
```

Comment:
This should use the constants from the output package

Reply:
Could you confirm if we should also do this for Progress and ConfigDir as well? If so, we will need to change the var names in output package for Progress to Capitalise and export